src/Microsoft.Health.SqlServer/Configs/SqlServerDataStoreConfiguration.cs

```diff
@@ -52,6 +53,7 @@ public class SqlServerDataStoreConfiguration
     /// <summary>
     /// If set, the client id of the managed identity to use when connecting to SQL, if AuthenticationType == ManagedIdentity.
     /// </summary>
+    [Obsolete("Update ConnectionString to supply User ID instead.")]
```

Comment:
User ID or managed identity?

Reply:
`User ID` as that is the field in the SQL connection string

Reply:
The connection strings should look like these: https://learn.microsoft.com/en-us/sql/connect/ado-net/sql/azure-active-directory-authentication?view=sql-server-ver16#using-active-directory-managed-identity-authentication